client/app/components/dialog-editor-modal-field-template/dynamic.html

```diff
@@ -0,0 +1,35 @@
+<!-- any dynamic element -->
+<div pf-form-group pf-label="{{'Entry Point'|translate}}">
+  <!-- TODO: until treepicker and API for it are in the SSUI -->
```

Comment:
@chriskacerguis @jntullo This might be another thing to implement in API. The Entry Point is displayed in the API, but not the data to the treepicker:

![screenshot from 2016-12-13 07-02-25](https://cloud.githubusercontent.com/assets/1187051/21129524/4c65dd4c-c102-11e6-8528-b3ee7332ae61.png)
![screenshot from 2016-12-13 07-02-03](https://cloud.githubusercontent.com/assets/1187051/21129525/4c65ede6-c102-11e6-9536-4131f655311e.png)